WooCommerce/UITestsFoundation/Screens/Orders/NewOrderScreen.swift

```diff
@@ -33,6 +37,10 @@ public final class NewOrderScreen: ScreenObject {
     ///
     private var addProductButton: XCUIElement { addProductButtonGetter(app) }
 
+    /// Add Customer Details button in the Customer Details section.
+    ///
```

Comment:
I have mixed feelings here; on the one hand, the New Order screen has a lot of intractable elements, so I feel more need for documentation here. On the other hand, I see what you mean about this seeming extraneous. 
I'd like to lean towards more documentation in general here, since devs may not be familiar with UI testing, and we're asking them to get more involved. 

I'll leave these in for now, but we can readjust as devs get more involved and we have a better sense for what documentation is most helpful.